cypress/automated-tests/youth-pass/address-section-required-fields.spec.js

```diff
@@ -4,8 +4,14 @@ describe('Youth Pass Address Section Required Fields', () => {
     it('proceeds through an application', () => {
         const applicantZipCode = '02114';
         const youthPassUrl = Cypress.env('youth_pass_url');
-        const randomBirthdate = faker.date.between('2003-11-02', '2009-11-01');
-        const applicantBirthdate = `${randomBirthdate.getMonth() + 1}/${randomBirthdate.getDate()}/${randomBirthdate.getFullYear()}`;
+        const todaysDate = new Date();
+        const twelveYearsAgo = todaysDate.getFullYear() - 12; 
+        const eighteenYearsAgo = todaysDate.getFullYear() - 18; 
+        const randomBirthdate = faker.date.between(`${eighteenYearsAgo}-11-02`, `${twelveYearsAgo}-11-01`);
```

Comment:
suggestion: A bunch of these constants and functions seem repeated across test files (and I could imagine them being useful for even more tests in the future. What would you think about consolidating them into a helpful file and tests could import whatever they needed?

Reply:
Love this suggestion!